### PR TITLE
Update gamestop.py

### PR DESCRIPTION
**Pull request by cjnunn**
Added a workaround for Gamestop login (Missing "MY ACCOUNT" link)

### CODE REVIEW

**Comment by aahurtado** [comment]
Is this for their desktop site? Their hamburger button menu doesn't have a Sign In option.

**Comment by cjnunn** [comment]
Yes, it is for desktop. I'm not sure why it isn't always there, but when the bot opens the site, it's there.

File: sites/gamestop.py [hunk 1]
```diff
@@ -56,8 +56,17 @@ def login(self):
...
+        # Workaround for missing "MY ACCOUNT" link
+        # Click hamburger icon
+        wait(self.browser, self.LONG_TIMEOUT).until(EC.element_to_be_clickable((By.CLASS_NAME, "hamburger-icon-redesign")))
+        self.browser.find_element_by_class_name('hamburger-icon-redesign').click()
+        # Click "Sign In"
+        wait(self.browser, self.LONG_TIMEOUT).until(EC.element_to_be_clickable((By.LINK_TEXT, "Sign In")))
+        self.browser.find_element_by_link_text('Sign In').click()
+        # End of workaround
 
         wait(self.browser, self.LONG_TIMEOUT).until(EC.element_to_be_clickable((By.ID, "signIn"))).click()
```
Comment (aahurtado) [inline sites/gamestop.py:72]: I ran the bot and Sign In doesn't exist under the hamburger menu. Add a check to see if it does exist before trying to click it. And if it does exist and we click it then we don't need to click Sign In again on line 71.

Capture

Comment (cjnunn) [reply sites/gamestop.py:72]: Hmm...That's strange. I can't get the site to load for me today...not sure if I got banned or not, but when I would run it, I didn't have the "My Account" or "Powerup Rewards" links at the top, but I had a "Sign In" option under the hamburger menu. 

Maybe for some reason it is keeping you logged in, and not me?

I don't really know PowerShell, but I'll see if I can add a check for it.

Comment (aahurtado) [reply sites/gamestop.py:72]: My Account is what it says by default i.e. when you're not logged in.

Comment (cjnunn) [reply sites/gamestop.py:72]: Looks like I'm IP banned. I switched to my hotspot, and the site loaded...but now, its showing the links. It wasn't showing that before.

I guess I'll add a check for the links, and if it's not there, I'll try the hamburger.

**Comment by cjnunn** [comment]
Disregard this request.

**Pull request closed without merging**